var/spack/repos/builtin/packages/ghostscript/package.py

```diff
@@ -29,14 +29,23 @@ class Ghostscript(Package):
     """An interpreter for the PostScript language and for PDF."""
 
     homepage = "http://ghostscript.com/"
-    url = "http://downloads.ghostscript.com/public/old-gs-releases/ghostscript-9.18.tar.gz"
 
+    version('9.21', '6f60d7fcb5eef6a8bec5abedf21c6a7008a8c0c7')
     version('9.18', '33a47567d7a591c00a253caddd12a88a')
 
     parallel = False
 
     depends_on('libtiff')
 
+    def url_for_version(self, version):
```

Comment:
Hi @citibeth, `version > 9.18` does seem to be the right criterion. Version 9.18 is available on both the old and new sites. Tarballs of earlier versions are only available on the old site, and new versions (including 9.19 and 9.20) are only available on github.

We could actually use github for both versions in this package (9.18 and 9.21). This would make the downloads more reliable, because the old site sometimes breaks the connection while downloading version 9.18. I have tried downloading from a number of independent networks, and they all had similar problems with the old site.

If I used the github URL for all supported versions (i.e. `>= 9.18`), would that cause any problems?